website/docs/docs/deploy/slim-ci-jobs.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Slim CI jobs in dbt Cloud"
```

Comment:
I feel like this needs a verb, otherwise it seems like another "About" page. 
```suggestion
title: "Set up Slim CI jobs in dbt Cloud"
```

Reply:
i struggled with this. i'm not a huge fan of Set up Slim CI jobs in dbt Cloud and Set up Slim CI jobs being on the same page. 

let's chat more! added this as a topic for discussion in our team meeting agenda